### PR TITLE
Added sampleapp source code

### PR DESCRIPTION
**Pull request by mukrishn**
## Type of change

- [ ] Refactor
- [x] New app
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Added node js sampleapp code, widely used in our control plane workload.
[slack source](https://redhat-internal.slack.com/archives/CNKHL6076/p1675347239027479?thread_ts=1656535268.898809&cid=CNKHL6076)

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by mukrishn** [comment]
@rsevilla87 PTAL, just added another webserver source code - sampleapp.

File: sampleapp/Containerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM node:7
```
Comment (krishvoor) [inline sampleapp/Containerfile:1]: Can you please bump the NodeJS version here?

**Review by krishvoor — COMMENTED** [review]
Let's bump the NodeJS version, also kindly rebase

**Review by rsevilla87 — APPROVED** [review]
lgtm!

**Pull request merged**